modules/capability/simulation/decoder_test.go

```diff
@@ -48,14 +48,14 @@ func TestDecodeStore(t *testing.T) {
 		{"other", ""},
 	}
 
-	for i, tt := range tests {
-		i, tt := i, tt
-		t.Run(tt.name, func(t *testing.T) {
+	for i, tc := range tests {
+		i, tc := i, tc
+		t.Run(tc.name, func(t *testing.T) {
 			switch i {
 			case len(tests) - 1:
 				require.Panics(t, func() { dec(kvPairs.Pairs[i], kvPairs.Pairs[i]) }, tt.name)
 			default:
-				require.Equal(t, tt.expectedLog, dec(kvPairs.Pairs[i], kvPairs.Pairs[i]), tt.name)
+				require.Equal(t, tc.expectedLog, dec(kvPairs.Pairs[i], kvPairs.Pairs[i]), tt.name)
```

Comment:
`tt.name` still referenced in `require` assertion, should be `tc` now

Reply:
fixed !